consumer_example.py

```diff
@@ -39,13 +39,15 @@
 
 # Override the process_chunk method to do useful things with the session data
 def process_chunk(client: Client, session: Session, chunk: Chunk) -> None:
-    bundles = list(client.read_chunk(chunk))
+    bundles = list(client.read_chunk(session.session_id, chunk.chunk_id))
 
-    client.create_card(
-        session.session_id,
-        "Python Consumer Example",
-        f"Chunk {chunk.chunk_id} has {len(bundles)} bundles",
-    )
+    logging.info(f"Chunk {chunk.chunk_id} has {len(bundles)} bundles")
+
+    # client.create_card(
```

Comment:
can the comment block be removed?

Reply:
I want to keep it in as a way to show how a call to create a card would look like. this is only relevant for the Recorder.